src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -322,7 +324,7 @@ private static bool CanUpdateReadOnlyProperty(Type propertyType)
         /// Creates suitable <see cref="object"/> for given <paramref name="bindingContext"/>.
         /// </summary>
         /// <param name="bindingContext">The <see cref="ModelBindingContext"/>.</param>
-        /// <returns>An <see cref="object"/> compatible with <see cref="ModelBindingContext.ModelType"/>.</returns>
+        /// <returns>An <see cref="object"/> compatible with <see cref="DefaultModelBindingContext.ModelType"/>.</returns>
```

Comment:
`ModelBindingContext.ModelType`. Need similar changes a couple of other places in this `class`.